src/main/java/com/wuest/prefab/structures/render/StructureRenderHandler.java

```diff
@@ -64,61 +68,66 @@ public static void setStructure(Structure structure, Direction assumedNorth, Str
         if (mc.world != null) {
             StructureRenderHandler.dimension = mc.world.getDimension().getLogicalHeight();
         }
+
+        if (structure != null && mc.world != null) {
+            renderSetup(mc.world, mc.player);
```

Comment:
Static function calls should be pre-pended with the class name to make it clearer who owns the static function.